CODEOWNERS

```diff
@@ -19,6 +19,7 @@
 /VERSION                                  @nrfconnect/ncs-code-owners
 /west.yml                                 @nrfconnect/ncs-code-owners @nrfconnect/ncs-merge
 /west-test.yml                            @nrfconnect/ncs-ci
+/.sonarcloud.properties                   @nrfconnect/ncs-ci
```

Comment:
this should be on line 14 to match Github Sort
![image](https://github.com/user-attachments/assets/25d3f940-4268-4ec8-a544-e870aa24ef8a)